src/odemis/gui/cont/tabs/sparc2_align_tab.py

```diff
@@ -205,44 +225,111 @@ def __init__(self, name, button, panel, main_frame, main_data):
         spot_stream.should_update.value = True
         self._spot_stream = spot_stream
 
+        self._blanker = None
```

Comment:
Please see line ~559 `if main_data.ebeam_blanker:` . If one was to read the code now, he'd be quite confused. Can you add a comment above this code block (line 559) that the "ebeam_blanker" and the `ebeam.blanker` are not the same thing. The ebeam_blanker is a dedicated hardware for fast blanking (and typically used for pulsed e-beam). The `ebeam.blanker` is just the standard (& slow) blanker.